lib/hanami/utils/class_attribute.rb

```diff
@@ -93,7 +96,7 @@ def inherited(subclass)
         # Class accessor for class attributes.
         # @api private
         def class_attributes
-          @class_attributes ||= Set.new
+          @class_attributes ||= Concurrent::Map.new
```

Comment:
This is still not thread safe.

Reply:
I was wondering if I had to move this attribute to other place like we do here https://github.com/hanami/utils/blob/master/lib/hanami/utils/string.rb#L77